common/templates/layouts/collection.njk

```diff
@@ -72,7 +72,7 @@
       {% block headerLinks %}
         <div class="govuk-grid-column-one-third">
           <ul class="govuk-list govuk-!-font-size-16 app-!-float-right app-print--hide">
-            {% if canAccess('moves:download') %}
+            {% if canAccess('moves:download') and context !== "allocations" %}
```

Comment:
maybe this is something that could be handled by the prohibitions stuff. If we can prohibit by locationType, why not be `context`?

Reply:
I looked at doing that but it's a bit of a rabbit hole at the moment. The app doesn't actually currently support downloading allocations either so I think we need to address this separately.